include/Display.h

```diff
@@ -52,28 +52,26 @@ class Display : public QOpenGLWidget{
 
     int getW() const;
     int getH() const;
-    const Document* getDocument() const;
+    Document* getDocument() const;
 	OrthographicCamera* getCamera() const;
 	DisplayManager* getDisplayManager() const;
+
+    Qt::MouseButton getRotateCamera() const;
+    Qt::MouseButton getMoveCamera() const;
+    Qt::KeyboardModifier getRotateAroundThirdAxisModifier() const;
```

Comment:
Remove these from the display class, but move the **rotateCameraMouseButton**, **moveCameraMouseButton**, and **rotateAroundThirdAxisModifier** members to MoveCameraMouseAction.